api/alationApi.js

```diff
@@ -1,114 +1,86 @@
-const axios = require('axios').default;
+import 'dotenv-defaults/config.js';
+import axios from 'axios';
+
+export const alationAxios = axios.create({
+	baseURL: encodeURI(`https://${process.env.ALATION_DOMAIN}/integration`),
+	headers: {
+		TOKEN: process.env.ALATION_API_ACCESS_TOKEN,
+	},
+});
 
 /**
- * Updates custom field, "Classification Matches", with classifiers
- * @param {String} alationDomain The domain of your Alation organization
- * @param {String} alationApiAccessToken The Alation API Access Token (short-term) 
- * @param {Array} objects The objects that contain data to update custom field
- * @returns JS Object
+ * Gets users in Alation.
+ *
+ * @async
+ * @returns {Promise<Boolean>} True || False
  */
-let putMultipleCustomFieldValues = async (alationDomain, alationApiAccessToken, objects) => {
-	const options = {
-		method: 'PUT',
-		url: encodeURI(`https://${alationDomain}/integration/v2/custom_field_value/`),
-		headers: { accept: 'application/json', 'content-type': 'application/json', TOKEN: alationApiAccessToken },
-		data: objects
-	};
-
+export const getUsers = async () => {
 	try {
-		const response = await axios.request(options);
-		return response.data;
+		await alationAxios.get(`/v1/user/?email=${process.env.ALATION_EMAIL}&limit=100&skip=0`);
+		return true;
 	} catch (error) {
+		console.error('GET alation users error');
 		if (error.response) {
-			console.error('PUT multiple custom field values error');
 			console.error(error.response.data);
 			console.error(error.response.status);
 		}
-		throw error;
+		return false;
 	}
 };
-exports.putMultipleCustomFieldValues = putMultipleCustomFieldValues;
```

Comment:
Where is putMultipleCustomFieldValues defined now?  Looks like it was deleted above (line 10)

Reply:
putMultipleCustomFieldValues was changed to updateMultipleCustomFieldValues to clarify what the function / API call is doing.
Custom field exists for each object after setting up custom field. Either manually or using createCustomField.js. We are just updating the values.